drivers/display/Kconfig.gc9x01x

```diff
@@ -0,0 +1,11 @@
+# Copyright 2023 Martin Kiepfer <mrmarteng@teleschirm.org>
+# SPDX-License-Identifier: Apache-2.0
+
+config GC9X01X
+	bool "GC9X01X display driver"
+	default y
+	depends on DT_HAS_GALAXYCORE_GC9X01X_ENABLED
+	select SPI
+	select PWM
```

Comment:
Is `select GPIO` missing here?

Reply:
Generally you are right, but I don't see it on the other SPI based drivers as well (ili, st). And they all utilize GPIOs as well.

Reply:
So, what would be the right approach then?